keps/sig-node/2400-node-swap/README.md

```diff
@@ -731,11 +849,11 @@ Here are specific improvements to be made:
 
 #### GA
 
-_(Tentative.)_
-
-- Test a wide variety of scenarios that may be affected by swap support.
-- Remove feature flag.
-- Remove the Swap Support using Burstable QoS Pods only deprecated in Beta 2.
+- Test a wide variety of scenarios that may be affected by swap support, including tests with aggressive memory stress.
+- Address memory-backed backed volumes which should not have access to swap.  
```

Comment:
I find this section a bit confusing. We "Address memory-backed volumes" in beta. It was required for GAing this feature so I think we should move this to Beta.

Reply:
> We "Address memory-backed volumes" in beta.

I don't believe this is correct.
This [was implemented](https://github.com/kubernetes/kubernetes/pull/124060) after Beta2 was released, so it merged in the time frame between Beta2 and GA. IOW, it doesn't fit as a criteria for Beta if Beta was released without it.

Am I missing something?

Reply:
This reads like you want to promote this as part of the GA promotion. We have done this in the beta phase and we are clear to GA because this is done.

Reply:
> This reads like you want to promote this as part of the GA promotion

But this is not written anywhere in the document.
What's written is that it's a requirement for GA graduation. I think it's obvious that it would be implemented before GA graduation then.

> We have done this in the beta phase

I'm not sure what you mean by that, I guess that's a matter of terminology?

This was implemented **after Beta was released**, therefore it doesn't make sense that it's a requirement for Beta graduation.

Maybe I'm completely off but this doesn't make sense to me

Reply:
Either way, now that the implementation history is updated, this document outlines elsewhere that it was implemented before swap graduated to GA.

Reply:
With the implementation history updated, this is a bit more clear. We have already done this work in between beta -> GA.

Sounds good to me.